src/pages/Signup.jsx

```diff
@@ -0,0 +1,19 @@
+import { SignupCard } from "../components/Signup"
+
+const Signup = () => {
+
+    return (
+        <div
+            style={{
```

Comment:
I think we should avoid using inline styling. So, see if you can make a class for this.

Reply:
as I want the `min-height` to be `100vh` always, I thought of using inline style for its priority over class or id selectors.